### ISSUE TITLE
[TalkBack in spanish] cannot change currency to something different than "dollar" when reading the "$" symbol

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#232]
```
What steps will reproduce the problem?
1. Talkback reads a bank account balance, i.e. $400
2. It reads "cuatro cientos DOLARES"
3. I cannot change what it reads when it sees the "$" symbol.

What is the expected output? What do you see instead?
It should say ""cuatro cientos PESOS". It reads "cuatro cientos DOLARES".

What version of the product are you using? On what operating system?
Talkback in a Samsung S3 Android 4.1.2 in spanish.

Thank you!
```

Original issue reported on code.google.com by `sebas_bs...@hotmail.com` on 20 Apr 2013 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
This is an ambiguity of the text to speech engine.  TalkBack does not and 
probably should not extrapolate such symbols before sending them to TTS.
```

Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:27
- Changed state: **WontFix**
